.github/workflows/release.yml

```diff
@@ -90,6 +90,12 @@ jobs:
           mv auth-arm64 auth
           tar -czvf auth-v$RELEASE_VERSION-arm64.tar.gz auth gotrue migrations/
 
+      - name: configure aws credentials for uploading release artifacts
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.PROD_AWS_ROLE }}
+          aws-region: us-east-1
+
```

Comment:
we only upload the release artifacts when the release-please PR is merged and not the rc versions right?

Reply:
ah didn't notice the `if` clause - should be fine then since the artifact will only be pushed when the release candidate is merged into master